# Wrong Division — Jane Street, November 2014

https://www.janestreet.com/puzzles/wrong-division-index/

![Wrong Division](https://www.janestreet.com/puzzles/Nov14.png)

Solved on paper.

Solution: 

## Step 1 

In [4]:
for i in range(6):
    print((2807 + 10 * i) * 7)

19649
19719
19789
19859
19929
19999


## Step 2 deductions

I ended up not getting a valid soution so went back over my work, and I made an error setting row 3 hundreds digit to 7 instead of 9!  After fixing that mistake, the rest of the solution propogated until I could just enumerate the 10 candidates below. 

In [8]:
for i in [1, 3, 5, 7, 9]:
    for j in [4, 5]:
        print(f"{i} , {j} = {(2807 + 10 * j) * (20710 + i)}")

1 , 4 = 58964217
1 , 5 = 59171327
3 , 4 = 58969911
3 , 5 = 59177041
5 , 4 = 58975605
5 , 5 = 59182755
7 , 4 = 58981299
7 , 5 = 59188469
9 , 4 = 58986993
9 , 5 = 59194183


We have a singluar soution where the ten-thousands place is 7 and the 10s place is 0.  So our dividend is $58975605$.
